plugin/files/airconsole/runtime.js

```diff
@@ -374,6 +384,42 @@ cr.plugins_.AirConsole = function(runtime)
     ret.set_string(this.ac_uid);
   };
 
+  Exps.prototype.ControllerDeviceIDs = function (ret)
+  {
+    var arr = this.air_console.getControllerDeviceIds();
+
+    // Ok, let's create a fake C2 JSON string that is compatible with C2 array load function
+    var jsonStr = new Object();
+    jsonStr['c2array'] = true;
+    jsonStr['size'] = [arr.length, 1, 1];
+    var data = [];
+    for (var i in arr) {
+      data.push([[arr[i]]]);
+    }
+    jsonStr['data'] = data;
```

Comment:
Yes, was planning to use it for scoreboard, and many other things. As I'm in a lean production, I never push further before the first things are agreed/accepted, got plenty of other things to do and no real time to restart all over again and again :) So if you like the idea, I'll implement that for other things